src/serviceWorker.js

```diff
@@ -28,6 +28,10 @@ export function waitForServiceWorker() {
  * @param {String} url The URL to prefetch
  */
 export async function prefetch(url) {
+  if (process.env.NODE_ENV !== 'production' && process.env.SERVICE_WORKER !== 'true') {
```

Comment:
Would it be more clear if the ENV var was named `PREFETCH`?

Reply:
We already have the SERVICE_WORKER environment variable which needs to be set to true to create a service worker in development, so I figured we can use it for this as well rather than adding another.  You generally only want prefetching when testing the service worker in development.